src/context/AuthorizerContext.jsx

```diff
@@ -0,0 +1,27 @@
+import React ,{useReducer ,createContext ,useContext }from 'react';
+import  authReducerFunc from '../reducer/authReducerFunc';
+
+
+
+const AuthorizerContext = createContext();
+
+const AuthorizeProvider = ({children})=>{
+    const initialAuthState = {
```

Comment:
Can define this `initialState` in respective reducer & import it here.
Also, can use the `initialState` in reducer for case: `LOGOUT`.

Reply:
ohk i will do that ,thank you